content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -231,187 +235,187 @@ and in the logs to monitor the state of the auditing subsystem.
+   ```shell
+   kube-apiserver ... --audit-policy-file=/etc/kubernetes/audit-policy.yaml --audit-log-path=/var/log/kube-audit --audit-log-format=json
+   ```
 
 1. Check audits for different namespaces in `/var/log/audit-*.log`
 
-### Use logstash to collect and distribute audit events from webhook backend
+### Use Logstash to collect and distribute audit events from webhook backend
 
 [Logstash](https://www.elastic.co/products/logstash)
 is an open source, server-side data processing tool. In this example,
 we will use logstash to collect audit events from webhook backend, and save events of
 different users into different files.
 
-1. install [logstash](https://www.elastic.co/guide/en/logstash/current/installing-logstash.html)
+1. install [Logstash](https://www.elastic.co/guide/en/logstash/current/installing-logstash.html)
 
 1. create config file for logstash
 
-    ```
-    cat <<EOF > /etc/logstash/config
-    input{
-        http{
-            #TODO, figure out a way to use kubeconfig file to authenticate to logstash
-            #https://www.elastic.co/guide/en/logstash/current/plugins-inputs-http.html#plugins-inputs-http-ssl
-            port=>8888
-        }
-    }
-    filter{
-        split{
-            # Webhook audit backend sends several events together with EventList
-            # split each event here.
-            field=>[items]
-            # We only need event subelement, remove others.
-            remove_field=>[headers, metadata, apiVersion, "@timestamp", kind, "@version", host]
-        }
-        mutate{
-            rename => {items=>event}
-        }
-    }
-    output{
-        file{
-            # Audit events from different users will be saved into different files.
-            path=>"/var/log/kube-audit-%{[event][user][username]}/audit"
-        }
-    }
-    EOF
-    ```
-
+   Write the following to, for example, `/etc/logstash/config`:
+   ```
+   input{
+       http{
+           #TODO, figure out a way to use kubeconfig file to authenticate to logstash
+           #https://www.elastic.co/guide/en/logstash/current/plugins-inputs-http.html#plugins-inputs-http-ssl
+           port=>8888
+       }
+   }
+   filter{
+       split{
+           # Webhook audit backend sends several events together with EventList
+           # split each event here.
+           field=>[items]
+           # We only need event subelement, remove others.
+           remove_field=>[headers, metadata, apiVersion, "@timestamp", kind, "@version", host]
+       }
+       mutate{
+           rename => {items=>event}
+       }
+   }
+   output{
+       file{
+           # Audit events from different users will be saved into different files.
+           path=>"/var/log/kube-audit-%{[event][user][username]}/audit"
+       }
+   }
+   EOF
+   ```
 1. start logstash
 
-    ```shell
-    bin/logstash -f /etc/logstash/config --path.settings /etc/logstash/
-    ```
+   ```shell
+   bin/logstash -f /etc/logstash/config --path.settings /etc/logstash/
```

Comment:
Are these instructions on the elastic docs site?
Could you link to the instructions instead of explaining in this page?

Reply:
> Are these instructions in the Elastic docs?

I didn't check - this PR is about tidying the existing page, so I have not been as thorough as I would if writing it.

I'll update to assume that `logstash` is in the reader's `$PATH`.